README.md

```diff
@@ -1,5 +1,7 @@
 # DevSecOps MVP
 
+**This repository contains some early experiments with DevOps tools and practices used by the AK DHSS. Current tools, standards and practices are now maintained in a private VSTS Git repo and is shared with vendors engaged on projects. `This repo is no longer actively maintained`.**

```

Comment:
Thinking about the statement "Current tools, standards and practices are now maintained in a private VSTS Git repo and is shared with vendors engaged on projects."  In concert with my own experience digging back into this repo occasionally for examples, I'm wondering if there is some set of needs that the new DevSecOps repo doesn't support.  A better library of starter examples for folks on the team who need them, and also some transparency for vendors considering bidding on future RFPs.  I think we should move forward with this PR, but wonder if we should chat briefly about this possible gap and whether/how to fill it going forward.

Reply:
@sztaylorakgov Agreed - I think it would be good to chat about this. I really like the idea of providing more resources for folks on the teams and also for interested vendors.

Merging this PR and creating a new PBI in VSTS for this work.